tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -12,6 +12,15 @@ parameters:
 
 # Set version
 steps:
+- task: Bash@3
+  name: InstallBuildTools
+  displayName: Install Fluid Build Tools
+  inputs:
+    targetType: 'inline'
+    workingDirectory: ${{ parameters.buildDirectory }}
+    script: |
+      npm install --global @fluidframework/build-tools@latest
```

Comment:
I could also install the version in local repo. As in install directly from the local FS.

Reply:
Are we worry about @latest?  this might cause unexpected change when the pipeline is run on older commit (e.g. patches)
Although it might make it a little more tedious to make changes that needs to apply to old branches as well.

Reply:
>I like install from NPM, but are we worry about @latest?

Yeah, I think FS install is better.

> Install from local FS would require us to build the tools ondemend no?

Yeah, I think so, but that still feels more maintainable than a separate file. Other ideas?

Reply:
You can still use a @ <build number>.  We just need to update here as well.

Alternative is to have a copy of the built script :D